windows-installer/fluent-template.conf

```diff
@@ -48,7 +48,7 @@
   max_retry_wait 300
   # Use multiple threads for processing.
   num_threads 8
-  # Use the gRPC transport.
+  # Temporarily disable gRPC to work around a gRPC bug (b/174475386).
```

Comment:
This doesn't tell users a whole lot. We should mention the transport that we _will_ use, and use the full URL of the public issue. Plus, we haven't really done performance comparisons, but gRPC is more expensive with structured data (jsonPayload), which most of Windows log messages are. So how about:
```
# Use REST transport, because gRPC is expensive for structured data
# and has a bug (http://issuetracker.google.com/174475386).
```
?

Reply:
Done